### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Aryan4884**
These comments provide a brief explanation of each step in the original code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,23 +41,25 @@ pip install neural-compressor
...
+
+# Import necessary libraries
 from neural_compressor.data import DataLoader, Datasets
 from neural_compressor.config import PostTrainingQuantConfig
 
+# Define the calibration dataset and create a data loader for it
 dataset = Datasets("tensorflow")["dummy"](shape=(1, 224, 224, 3))
 dataloader = DataLoader(framework="tensorflow", dataset=dataset)
 
-from neural_compressor.quantization import fit
```
Comment (chensuyue) [inline README.md:54]: This line shouldn't be removed.

File: README.md [hunk 2]
```diff
@@ -41,23 +41,25 @@ pip install neural-compressor
 # Install Intel Neural Compressor and TensorFlow
 pip install neural-compressor
 pip install tensorflow
-# Prepare fp32 model
+
+# Prepare the FP32 model by downloading it
 wget https://storage.googleapis.com/intel-optimized-tensorflow/models/v1_6/mobilenet_v1_1.0_224_frozen.pb
-```
-```python
```
Comment (chensuyue) [inline README.md:47]: This example including 2 part, 1st part is bash cmd, use to prepare env, 2nd part is python code need to run with python, so they are not the same level code, should not use the same block.

File: README.md [hunk 3]
```diff
@@ -41,23 +41,25 @@ pip install neural-compressor
...
 from neural_compressor.data import DataLoader, Datasets
 from neural_compressor.config import PostTrainingQuantConfig
 
+# Define the calibration dataset and create a data loader for it
 dataset = Datasets("tensorflow")["dummy"](shape=(1, 224, 224, 3))
 dataloader = DataLoader(framework="tensorflow", dataset=dataset)
 
-from neural_compressor.quantization import fit
-
+# Perform post-training quantization on the FP32 model
```
Comment (chensuyue) [inline README.md:56]: The comments LGTM.

**Comment by chensuyue** [comment]
@Aryan4884 would you like to enhance this PR? or I will close it and update your comments in a new PR.

**Comment by Aryan4884** [comment]
you can update it into a new PR.Thanks for your suggestion.

**Pull request closed without merging**